src/reaction_network.jl

```diff
@@ -407,12 +411,13 @@ function make_reaction_system(ex::Expr; name = :(gensym(:ReactionSystem)))
     # Returns the rephrased expression.
     quote
         $ps
+        $ns_ps
         $ivexpr
         $vars
         $sps
 
         Catalyst.make_ReactionSystem_internal($rxexprs, $tiv, union($spssym, $varssym),
-                                              $pssym; name = $name,
+                                              union($pssym, $ns_pssym); name = $name,
```

Comment:
Can we avoid a union here in the very common case that no noise scaling is given? i.e. define a variable that is the `ps` if there is no noise scaling, and the union otherwise, and then pass it in here.